server/pulp/server/webservices/views/root_actions.py

```diff
@@ -0,0 +1,27 @@
+from django.views.generic import View
+
+from pulp.server.auth import authorization
+from pulp.server.managers import factory
+from pulp.server.webservices.controllers.decorators import auth_required
+from pulp.server.webservices.views.util import generate_json_response
+
+
+class LoginView(View):
+    """
+    View for API root-level actions.
+    """
+
+    @auth_required(authorization.READ)
+    def post(self, request, *args, **kwargs):
+        """
+        Return client SSL certificate and a private key
+
+        :param request: WSGI request object
+        :type request : WSGIRequest
```

Comment:
It was requested that we include the full path to this object since we do not import it.

`django.core.handlers.wsgi.WSGIRequest`